### ISSUE TITLE
[Question]: Embedding with fine tuned classifier model 

### ISSUE THREAD

**Issue by 8Raouf24** [flairNLP/flair#3430]
### Question

I'm using the following code to train a Hierarchical multi-label classification  model : 
`    with open(data_dir + 'corpus.pkl', 'rb') as handle:
        corpus = pickle.load(handle)

    current_model = f"{PREFIX}__{model_version}"

    label_dict = corpus.make_label_dictionary(label_type="labels")
    label_dict.multi_label = True

    logger.info("Flair Model init")

    document_embeddings = TransformerDocumentEmbeddings(
        'camembert/camembert-large',
        allow_long_sentences=True,
        fine_tune=True,
        is_token_embedding=True,
        use_context=True,
        model_max_length=512)

    classifier = TextClassifier(
        document_embeddings,
        label_dictionary=label_dict,
        multi_label=True,
        label_type="labels")

    trainer = ModelTrainer(classifier, corpus)

    logger.info("Launching Training")
    trainer.fine_tune(
        model_dir,
        use_final_model_for_eval=False,
        max_epochs=num_epochs,
        learning_rate=1e-6,
        min_learning_rate=1e-8,
        write_weights=True,
        mini_batch_chunk_size=8,
        mini_batch_size=32,
        checkpoint=True,
        shuffle=False,
        create_file_logs=True,
        optimizer=torch.optim.AdamW,
        embeddings_storage_mode="gpu",
        use_tensorboard=True,
        tensorboard_log_dir=model_dir)`
        
        I was wondering if it was possible to use the resulted model not only for classification but also exploit the fine-tuned camembert model to create document embeddings.
        
 Thanks in advance

**Comment by helpmefindaname** [comment]
Hi @8Raouf24 yes you can,
if you pass `embedding_storage_mode="gpu"` to the [model.predict()](https://flairnlp.github.io/flair/v0.13.1/api/embeddings/transformer.html#flair.embeddings.transformer.TransformerEmbeddings) function, you can also access `sentence.embedding` of the sentences that you predicted.
